Quaver.Shared/Screens/Gameplay/UI/Bubble/HitBubbles.cs

```diff
@@ -20,31 +20,41 @@ public class HitBubbles : Container
     private int _maxBubbleCount;
     private readonly HitBubblesType _hitBubblesType;
     private AnimatableSprite _background;
-    private readonly float _hitBubblesScale;
     private readonly float _hitBubblePadding;
     private readonly float _borderPadding;
-    private bool ShowHitBubbles => SkinKeys.HitBubblesScale == 0 || !ConfigManager.DisplayHitBubbles.Value;
+    private GameplayScreen Screen { get; }
+
+    private bool HideHitBubbles =>
+        (Screen?.IsCalibratingOffset ?? false)
+        || SkinKeys.HitBubblesScale == 0
+        || !ConfigManager.DisplayHitBubbles.Value;
+
+    private HitBubbleRecordedJudgement RecordedJudgements =>
+        Screen.InReplayMode
+            ? HitBubbleRecordedJudgement.All
+            : SkinKeys.HitBubblesRecordedJudgements;
+
```

Comment:
Maybe check if the Autoplay mod is active here instead of replay mode, since we don't want to display all in tournaments or regular replays.

Reply:
Changed the logic. It still has replay mode check but with constraint that we are playtesting, since I still want All to apply when we are previewing.